nle/nethack/actions.py

```diff
@@ -15,11 +15,11 @@ def C(c):
 
 
 class TextCharacters(enum.IntEnum):
-    PLUS = ord("+")
+    PLUS = ord("+")  # Also SEESPELLS.
```

Comment:
Seems silly to have the same actions twice. Anyone looking to "lookup" a key press will have trouble. Much cleaner to make sure the mapping is one-to-one. I would remove here.

Reply:
The lookup dict deals with that. I guess for symmetry reasons I found it nicer to have `PLUS` available given that `MINUS` is. I'm fine with either.

Reply:
(Removed action changes from PR.)